app/bundles/PageBundle/EventListener/CampaignSubscriber.php

```diff
@@ -91,7 +107,8 @@ public function onCampaignBuild(CampaignBuilderEvent $event)
         $pageHitTrigger = [
             'label'          => 'mautic.page.campaign.event.pagehit',
             'description'    => 'mautic.page.campaign.event.pagehit_descr',
-            'formType'       => 'campaignevent_pagehit',
+            'formAlias'      => 'campaignevent_pagehit',
+            'formType'       => CampaignEventPageHitType::class,
```

Comment:
This change should happen only for ConfigSubscribers. All other subscribers should only replace the alias string with the class name. There are other places with this issue in this PR.

Reply:
Thank you! Fixed